CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to become a contributor and submit your own code
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
+1. Break your changes in to a series of logical self-contained commits.
+1. Include release notes files when appropriate following the guidelines
+   in [Neptune](https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst).
+1. Submit your pull request.
+
+## Reviewing and Rebasing and Merging Patches
+
+1. All feedback should be professionally given and non-personal. Offensive,
+   intimidating, insulting, racist, homophobic, and sexist comments and
+   behavior will not be tolerated.
+1. Reviewers will comment on your code providing feedback to point out any
+   problems and improve the quality of the code. Feel free to respond to these
+   comments if you have any questions to clarify the meaning of the comments.
+1. Please try and pay attention to all the comments and address them as it is
+   frustrating for reviewers to review a PR multiple times and find the same
+   issues.
```

Comment:
Maybe remove this line "it is frustrating for reviewers ... " as it might not match the tone of the entire guide (although it is ;-) )

Reply:
I could put something more formal if you like:
`it wastes the time and resources of the project to review a PR multiple times and find the same issues.`?

Reply:
I would just go for "Please try and pay attention to all the comments and try to address them all", but I don't have a strong opinion.